server/sv_main.lua

```diff
@@ -24,45 +30,92 @@ RegisterNetEvent("ps-camera:CreatePhoto", function(url)
 end)
 
 RegisterNetEvent("ps-camera:savePhoto", function(url, streetName)
+    
     local source = source
     local player = QBCore.Functions.GetPlayer(source)
     if not player then return end
 
     local location = streetName
 
     local info = {
-        image = url,
+        ps_image = url,
         location = location
     }
-    player.Functions.AddItem("photo", 1, nil, info)
-    TriggerClientEvent('inventory:client:ItemBox', source, QBCore.Shared.Items['photo'], "add")
+    if not (Config.Inv == "qb" or Config.Inv == "ox") then return print('^1[Error] Your Config.Inv isnt set.. you probably had a typo\nYou have it set as= Config.Inv = "'.. Config.Inv .. '"') end
+    
```

Comment:
```suggestion
if not (Config.Inv == "qb" or Config.Inv == "ox") then 
    ConfigInvInvalid()
    return;
end
```

and then extract the print to a new local function.